sinch/domains/voice/models/calls/responses.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+from sinch.domains.voice.models import Price, Destination
+
+
+@dataclass
+class GetVoiceCallResponse(SinchBaseModel):
+    from_: str
+    to: Destination
+    domain: str
```

Comment:
Candidate for a literal

Reply:
I can add it here, but this is a `response` kind of object. Maybe leaving this open will be a better (more defensive) API handling strategy for us?

Reply:
Can you do something like `Literal["domain_a"] | str` to be safe but still provide some guide for possible values?